pg_lakehouse/src/fdw/base.rs

```diff
@@ -78,9 +78,6 @@ pub trait BaseFdw {
             handler,
         )?;
 
-        // Ensure we are in the same DuckDB schema as the Postgres schema
-        connection::execute(format!("SET SCHEMA '{schema_name}'").as_str(), [])?;
```

Comment:
If my understanding is correct, the code here is to set the duckdb schema for the exector hook. The code here can be replaced by the set_search_path.